### PR TITLE
Feature - Switch host to mainnet & update styling

### PR DESCRIPTION
**Pull request by bradleystachurski**
This PR updates the host from `http://127.0.0.1:27592 ` to `https://viewer.mainnet.sparkswap.com:27592` and adds analytics using `segment.io`. There are also a few styling updates:
- Formatted numbers in orderbook (grayed out after last significant digit)
- More prominent Call To Action (bold black and open to changing the color)
- Icons next to links at bottom

Before
---
image

After
---
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -114,9 +211,22 @@ class App extends Component {
...
+              <a href="https://github.com/sparkswap" className="link-item">
+                <Icon name="github" />
+                GitHub
+              </a>
+              <a href="https://sparkswap.com/onboarding" className="link-item">
+                <Icon name="life ring" />
+                Get Help
+              </a>
+              <a href="#" onClick={this.showMailingPopUp} className="link-item">
+                <Icon name="newspaper" />
```
Comment (treygriffith) [inline src/App.js]: How about a mail icon?

File: src/App.js [hunk 2]
```diff
@@ -114,9 +211,22 @@ class App extends Component {
...
             </div>
             <div className="links">
-              <a href="https://sparkswap.com/chat">Chat</a>
-              <a href="https://sparkswap.com/onboarding">Get Help</a>
-              <a href="#" onClick={this.showMailingPopUp}>Subscribe to Updates</a>
+              <a href="https://sparkswap.com/chat" className="link-item">
+                <Icon name="discord" />
+                Chat
+              </a>
+              <a href="https://github.com/sparkswap" className="link-item">
```
Comment (treygriffith) [inline src/App.js]: Our docs site might be better to link than GitHub.

Comment (bradleystachurski) [reply src/App.js]: Sounds good, I'll look for some icons that fit with docs.

File: src/App.css [hunk 3]
```diff
@@ -59,11 +59,27 @@
...
 .App .Footer a {
   color: inherit;
   text-decoration: inherit;
 }
+
+.App .green-text {
+  color: green;
+}
+
+.App .red-text {
```
Comment (treygriffith) [inline src/App.css]: better to call this `ask` or similar rather than `red-text`. It's more flexible.

File: src/App.js [hunk 4]
```diff
@@ -22,7 +32,7 @@ class App extends Component {
         name: 'BTC/LTC'
       }
     ]
-    this.host = 'http://127.0.0.1:27592'
+    this.host = 'https://viewer.mainnet.sparkswap.com:27592'
```
Comment (treygriffith) [inline src/App.js:35]: since you define the host here, you may want to define the username and password here as well.

File: public/index.html [hunk 5]
```diff
@@ -21,6 +21,14 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
+    <!-- Analytics -->
+    <script>
+      !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
+        analytics.load("ZGOnMWY4A5epIqq8RBpwQxoHfztNrslW");
+        // analytics.page();
```
Comment (treygriffith) [inline public/index.html]: Is this meant to be in this commit?

File: src/App.js [hunk 6]
```diff
@@ -73,7 +195,7 @@ class App extends Component {
             </a>
             <div className="Install">
               <a href="https://sparkswap.com/docs/getting-started">
-                <Button size="large" color='grey'>Install Now</Button>
+                <Button size="large" color='black'>Install Now</Button>
```
Comment (treygriffith) [inline src/App.js]: can you add a track link to this one?

File: src/App.js [hunk 7]
```diff
@@ -22,18 +32,28 @@ class App extends Component {
...
+
+    // This is hardcoded for demo purposes only
+    this.username = 'sparkswap'
+    this.password = 'sparkswap'
   }
 
   componentDidMount () {
-    this.marketData = new MarketData(this.markets[0].name, this.host)
+    // Analytics
+    window.analytics.page('Home')
```
Comment (treygriffith) [inline src/App.js]: Is `Home` the right thing here? Maybe `Orderbook`?

File: public/index.html (hunk 5 above)
Comment (bradleystachurski) [reply public/index.html]: I was following the installation steps from (https://github.com/segmentio/analytics-react#%EF%B8%8F-step-1-copy-the-snippet), but we can rip it.

**Review by treygriffith — APPROVED** [review]

**Comment by bradleystachurski** [comment]
[Web GUI for MainNet orderbook](https://trello.com/c/dNvjmabd/1321-web-gui-for-mainnet-orderbook)

**Pull request merged**